packages/lab/src/stepped-tracker/stepReducer.ts

```diff
@@ -0,0 +1,171 @@
+import type { Step } from "./Step";
+import { assignSteps, autoStageSteps, flattenSteps, resetSteps } from "./utils";
+
+export namespace StepReducer {
+  export interface State {
+    steps: Step.Props[];
+    flatSteps: Step.Props[];
+    activeStep: Step.Props | null;
+    previousStep: Step.Props | null;
+    nextStep: Step.Props | null;
+    activeStepIndex: number;
+    started: boolean;
```

Comment:
started/ended are languages to be aligned with design

Reply:
Can you provide more context, please on what specifically I need to discuss with design.

Reply:
both design and dev need to be using the same name for things and if we have used equivalent language before on another component it needs to be re-used

Reply:
For more context, although this is a code-only name in hook, it has high likelihood appearing in a component guidance or pattern doc (given this is used in example code).